### DISCUSSION TITLE
PWAsForFirefox 2.7.1

### DISCUSSION THREAD

**Discussion by filips123** [filips123/PWAsForFirefox#367] [Announcements]
## Fixed

* **Prevented syncing some preferences and reset problematic ones:**
  Web app profiles now prevent syncing certain preferences that are commonly set to different values in web apps, as well as those known to cause problems. If you know what you are doing and still want to sync one of those preferences, you can enable them back using the `services.sync.prefs.sync.` control preferences. In the future, we could try to implement a different syncing "channel" just for web apps.
  Additionally, the preferences known to cause problems were reset. Those preferences currently cannot be changed by users as they will be reset on every web app startup. This might be removed in the future, once enough users have had those preferences reset.
  This is another attempt in fixing issues https://github.com/filips123/PWAsForFirefox/issues/253 and https://github.com/filips123/PWAsForFirefox/issues/349.

* Added missing `$pkgdir` in the destination path of AUR package.

<hr /><em>This discussion was created from the release <a href='https://github.com/filips123/PWAsForFirefox/releases/tag/v2.7.1'>PWAsForFirefox 2.7.1</a>.</em>

**Comment by Inrumpo** [comment]
I do get the update prompt in my Firefox browser (115.0.2),

> The native connector is currently outdated. It is compatible with the current extension version, but it is strongly recommended to update it to get the latest features and fixes. Please follow the instructions below. 

but unfortunately, clicking on those links below doesn't do anything. This is happening on my two Windows machines.

What's the alternate route to install this update?

**Reply by filips123** [reply]
I can reproduce this, but only in a production build, that's why I didn't catch it earlier... This is very weird and probably a problem with the build tool that I use, maybe caused by [recently updated dependencies](https://github.com/filips123/PWAsForFirefox/commit/6a049b28e6d2ae1a3e663a4f1d195ba75a9f3db0).

It's quite an important problem. I hope the only thing it affects is the installation instructions page, but considering it seems to be a problem with a build tool, it might affect even more things in the extension. I now disabled versions 2.7.0 and 2.7.1, and you should downgrade the extension to 2.6.1. I hope I'll be able to fix this properly as soon as possible.

I created issue #368 about this problem.
